Engine/ac/button.cpp

```diff
@@ -477,6 +549,14 @@ void RegisterButtonAPI()
         { "Button::SetText^1",            API_FN_PAIR(Button_SetText) },
         { "Button::get_TextAlignment",    API_FN_PAIR(Button_GetTextAlignment) },
         { "Button::set_TextAlignment",    API_FN_PAIR(Button_SetTextAlignment) },
+        { "Button::get_WrapText",         API_FN_PAIR(Button_GetWrapText) },
+        { "Button::set_WrapText",         API_FN_PAIR(Button_SetWrapText) },
+        { "Button::get_TextHorizontalPadding", API_FN_PAIR(Button_GetTextHorizontalPadding) },
+        { "Button::set_TextHorizontalPadding", API_FN_PAIR(Button_SetTextHorizontalPadding) },
+        { "Button::get_TextVerticalPadding", API_FN_PAIR(Button_GetTextVerticalPadding) },
+        { "Button::set_TextVerticalPadding", API_FN_PAIR(Button_SetTextVerticalPadding) },
+        { "Button::get_WrapText",         API_FN_PAIR(Button_GetWrapText) },
```

Comment:
I thin these two `"Button::get_WrapText"` and `"Button::set_WrapText"` are duplicate?